### PR TITLE
[Torch Fx] PTQ Quantizer config test

### PR DESCRIPTION
**Pull request by rk119**
### Changes

New test file added in tests/torch/fx. Implemented TemplateTestQuantizerConfig for the ptq quantizer for TorchFX backend.

### Related tickets

Closes issue #2874

### CODE REVIEW

File: tests/torch/fx/test_quantizer_config.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from tests.torch.ptq.helpers import get_sum_aggregation_nncf_graph
+
+
+class TestQuantizerConfig(TemplateTestQuantizerConfig):
+    def get_algo_backend(self):
+        return FXMinMaxAlgoBackend()
+
+    @pytest.fixture
+    def single_conv_nncf_graph(self) -> NNCFGraphToTest:
+        return get_single_conv_nncf_graph()
```
Comment (daniil-lyakhov) [inline tests/torch/fx/test_quantizer_config.py:30]: Please do not use pytorch nncf grpahs, please create the same functions in torchFX test helpers as it done here https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/ptq/helpers.py but do not specify the layer attributes as TorchFX nncf graph does not require layer attributes

**Comment by mlukasze** [comment]
hey @rk119 do you need any advice how to fix tests?

File: tests/torch/fx/test_quantizer_config.py (hunk 1 above)
Comment (rk119) [reply tests/torch/fx/test_quantizer_config.py:30]: > Please do not use pytorch nncf grpahs, please create the same functions in torchFX test helpers as it done here https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/ptq/helpers.py but do not specify the layer attributes as TorchFX nncf graph does not require layer attributes

Alright yes. Thank you for your guidance.

**Comment by rk119** [comment]
> hey @rk119 do you need any advice how to fix tests?

Hi, I apologize for delaying this as I was busy with other commitments. I have updated the files according to the changes requested by @daniil-lyakhov. Let me know if there is anything else I should look into or update.

**Review by daniil-lyakhov — APPROVED** [review]
LGTM, thanks for the contribution!

**Comment by daniil-lyakhov** [comment]
@AlexanderDokuchaev, please merge

**Pull request merged**